src/testIntegration/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/magiclink/MagicLinkResourceTest.kt

```diff
@@ -122,15 +122,21 @@ class MagicLinkResourceTest(
         .expectStatus().isCreated
 
       val savedSecret = magicLinkSecretRepository.findAll().firstOrNull()
-      val message = mailCatcherWebClient.get()
+      val messageJson = mailCatcherWebClient.get()
         .uri("/messages/1.json")
         .accept(MediaType.APPLICATION_JSON)
         .retrieve()
-        .bodyToMono(Message::class.java)
+        .bodyToMono(MessageJson::class.java)
+        .block()
+      val messageSource = mailCatcherWebClient.get()
+        .uri("/messages/1.source")
+        .accept(MediaType.APPLICATION_JSON)
+        .retrieve()
+        .bodyToMono(String::class.java)
         .block()
 
-      assertThat(message?.recipients).containsExactly("<some.email@company.com.cjsm.net>")
-      assertThat(message?.source).contains("${magicLinkConfig.url}?secret=${savedSecret?.secretValue}")
+      assertThat(messageJson?.recipients).containsExactly("<some.email@company.com.cjsm.net>")
+      assertThat(messageSource).contains("${magicLinkConfig.url}?secret=${savedSecret?.secretValue}")
```

Comment:
A breaking change to the mailcatcher API [from over 3 years ago (!!!)](https://github.com/sj26/mailcatcher/pull/390) has only just made it into the docker image we use, and this broke a few tests.